### PR TITLE
Implementing changes

### PR DESCRIPTION
**Pull request by MonikaFeigler**
1. Ecommerce template - adding new destination - google sheet for data studio dashboard
2. updating profile selection in google analytics templates
3. adding 2 new inputs from and to date to google analytics extractor in google analytics template
4. updating inputs for google sheet destinations in all templates where google sheet is used
5. renaming extractor to data sources and writer to data destination

### CODE REVIEW

File: keboola-ecommerce-shoptet/v0/src/inputs.jsonnet [hunk 1]
```diff
@@ -2,10 +2,12 @@ local snowflake = import "/<common>/out-ecommerce-snowflake/v0/src/inputs.jsonne
 local googlesheet = import "/<common>/out-ecommerce-gsheet/v0/src/inputs.jsonnet";
 local shoptet = import "/<common>/in-ecommerce-shoptet/v0/src/inputs.jsonnet";
 local bigquery = import "/<common>/out-ecommerce-bigquery/v0/src/inputs.jsonnet";
-local googlesheet = import "/<common>/googlesheet_inputs.jsonnet";
+local googlesheet = import "/<common>/out-ecommerce-gsheet/v0/src/inputs.jsonnet";
```
Comment (webrouse) [inline keboola-ecommerce-shoptet/v0/src/inputs.jsonnet:5]: Is the file `googlesheet_inputs.jsonnet` no longer used?
If so, please delete it.

Comment (MonikaFeigler) [reply keboola-ecommerce-shoptet/v0/src/inputs.jsonnet:5]: it's done, I deleted the unused files.

**Review by webrouse — APPROVED** [review]
LGTM

I went through it quickly, and I didn't find a problem.
I did not try individual templates.

**Comment by webrouse** [comment]
@MonikaFeigler , merged, please write a brief summary of what you changed to Slack `#kbc-news-feed` . 
So private beta testers will be not surprised.

**Pull request merged**